clients/privacy-center/packages/fides-consent/src/lib/consent-banner.ts

```diff
@@ -0,0 +1,553 @@
+import {
```

Comment:
Reviewer note: this is a big file, but the most important function is at the very bottom: `initBanner`. That's the code that goes in, checks to see if the banner should be initialized, and then builds the DOM elements and adds to the document.